test_util/aws.py

```diff
@@ -424,16 +424,30 @@ class VpcCfStack(CfStack):
     @classmethod
     def create(cls, stack_name, instance_type, instance_os, instance_count,
                admin_location, key_pair_name, boto_wrapper):
-        ami_code = OS_AMIS[instance_os][boto_wrapper.region]
+        stack = cls.create_from_ami(
+            stack_name,
+            instance_type,
+            OS_AMIS[instance_os][boto_wrapper.region],
+            instance_count,
+            admin_location,
+            key_pair_name,
+            boto_wrapper,
+        )
+        return stack, OS_SSH_INFO[instance_os]
+
+    @classmethod
+    def create_from_ami(cls, stack_name, instance_type, instance_ami, instance_count,
```

Comment:
[minor] - In general, it might be a good idea to write dependency (this method) above the method from it is used (`create`).